tst/WebRTCClientTestFixture.h

```diff
@@ -1,6 +1,7 @@
 #include "gtest/gtest.h"
 #include "../src/source/Include_i.h"
 #include <memory>
+#include <string>
```

Comment:
I noticed that some tests were using std::string, but the header was not included here or in the tests.